File: Python/Python - EASY/PART - 002/Top Scoring Student.py
#Your code below
n = int(input())

maxTotal = 0
maxStudent = ""

for i in range(n):
    inp = input().split(":")
    
    total = sum(map(int, inp[1:]))
    
    if total > maxTotal:
        maxTotal =  total
        maxStudent = inp[0]
        
print(maxStudent)

File: Java/Java - Easy/Part - 002/Top Scoring Student.java
import java.util.*;
public class Hello {

    public static void main(String[] args) {
		//Your Code Here
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.nextLine();
        int max = 0;
        String maxStudent= "";
        for(int i=1;i<=n;i++){
            String[] input = sc.nextLine().split(":");
            int sum = 0;
            for(int j=1;j<=3;j++){
                sum += Integer.parseInt(input[j]);
            }
            if(sum > max){
                max = sum;
                maxStudent = input[0];
            }
        }
        System.out.println(maxStudent);
	}
}